commands/template_store_pull.go

```diff
@@ -0,0 +1,61 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultTemplateList = "https://raw.githubusercontent.com/martindekov/store/martindekov/unofficial_templates_json/templates.json"
+)
+
+func init() {
+	pull, _, _ := templateCmd.Find([]string{"pull"})
+	templateStoreCmd.Flags().AddFlagSet(pull.Flags())
+
+	templateStoreCmd.AddCommand(templateStorePullCmd)
+}
+
+var templateStorePullCmd = &cobra.Command{
+	Use:   `pull [TEMPLATE_NAME]`,
+	Short: `Pull templates from store`,
```

Comment:
I assume we have a flag to override the store json file location right?

Reply:
Added the `--url` flag here also ✔️